src/main/java/HashTable/HashTable.java

```diff
@@ -0,0 +1,40 @@
+package HashTable;
+
+
+import java.util.Hashtable;
+
+public class HashTable {
+//    Creation of Hash Table
+    Hashtable<Integer, String> hashTable = new Hashtable<>();

```

Comment:
You're creating a HashTable class which should be an Array of Linked Lists under the hood. You would then do some type of serialization to create a key which points to an index in that array.